src/routes/upcoming.jsx

```diff
@@ -40,6 +36,18 @@ const handleDelete = (id) => {
       console.log(err.message);
     }
   };
+  
+  const handleDelete = (id) => {
```

Comment:
Usually best practice to declare functions like this: 

function handleDelete(id) {
...
}

Helps with debugging.